client/src/components/SharedComponents/Signup/signup.css

```diff
@@ -92,6 +92,6 @@ button:hover {
   color: red;
 }
 
-.TrueValidate {
+.True-vsalidate {
```

Comment:
you have a typo here in ```validate```
```suggestion
.true-validate {
```

Reply:
@FarahZaqout  
Done!